scripts/build.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+#
+# This script is used to build releases for multiple platforms.
+# * Ensure you have added $GOPATH/bin to $PATH
+# 1. Checkout the release tag: # git checkout <tag>
+# 2. Run this script: ./scripts/build.sh
+
+if [[ ! -d "./scripts" ]]; then
+  echo "ERROR: scripts not found in pwd."
+  echo "Please run this from the root of the terraform provider repository"
+  exit 1
+fi
+
+GIT_TAG=$(git describe --tags)
+
+# The arch/os
+XC_ARCH=${XC_ARCH:-"amd64"}
+XC_OS=${XC_OS:-linux darwin windows}
+XC_EXCLUDE_OSARCH="!darwin/arm !darwin/386"
+
+# Delete the old dir
+echo "==> Cleaning up..."
+rm -f bin/* && rm -rf pkg/* && mkdir -p bin/
+
+# If required for only current platform set LOCAL=1
+if [ "${LOCAL}x" != "x" ]; then
+    XC_OS=$(go env GOOS)
+    XC_ARCH=$(go env GOARCH)
+fi
+
+if ! which gox > /dev/null; then
+    echo "==> Installing gox..."
+    go get -u github.com/mitchellh/gox
```

Comment:
this will add `gox` (and `github.com/hashicorp/go-version`) to `go.mod`. Could you add `gox` to `tools.go` instead?

Reply:
I think now we just need `go install` here now to use the locally vendored version and prevent this script to making changes to go.mod

Reply:
also please run `go mod tidy && go mod vendor` and include the changes in this commit.

Reply:
@LorbusChris Sorry for the delay.

I have made it `go install` and also `go mod tidy && go mod vendor` did not have any change.

Reply:
Ah you meant to add `go mod tidy && go mod vendor` in the script.. done!